gotestskip.go

```diff
@@ -100,10 +111,11 @@ func mainErr() error {
 		}
 		return nil
 	}
-	skip, err := readSkipConfig(skipFile)
+	skip, err := readSkipConfig(skipFile, os.Getenv("GO_SKIP_ROOT"))
 	if err != nil {
 		return err
 	}
+	fmt.Printf("%#v", skip)
```

Comment:
This should probably be removed before merging.

Reply:
done